data.md

```diff
@@ -137,12 +137,15 @@ The `#chop` function will ensure that an integer value is wrapped to the correct
 ```k
     syntax IVal ::= #chop ( IVal ) [function]
  // -----------------------------------------
-    rule #chop(< ITYPE > N) => < ITYPE > (N modInt #pow(ITYPE)) [concrete]
+    rule #chop(< ITYPE > N) => < ITYPE > N                      requires #inUnsignedRange(ITYPE, N)
```

Comment:
These extra rules help the solver to go through without using modulus, when possible.